{"name": "CVE-2016-7942", "public_date": "2016-09-25T00:00:00", "package_state": [{"fix_state": "Will not fix", "product_name": "Red Hat Enterprise Linux 5", "package_name": "libX11", "cpe": "cpe:/o:redhat:enterprise_linux:5"}, {"fix_state": "Will not fix", "product_name": "Red Hat Enterprise Linux 6", "package_name": "libX11", "cpe": "cpe:/o:redhat:enterprise_linux:6"}, {"fix_state": "Will not fix", "product_name": "Red Hat Enterprise Linux 7", "package_name": "libX11", "cpe": "cpe:/o:redhat:enterprise_linux:7"}], "document_distribution": "Copyright \u00a9 2016 Red Hat, Inc. All rights reserved.", "cvss3": {"status": "draft", "cvss3_scoring_vector": "CVSS:3.0/AV:A/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:L", "cvss3_base_score": "3.1"}, "threat_severity": "Low", "bugzilla": {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1381859", "description": "\nCVE-2016-7942 libX11: Insufficient validation of server responses in XGetImage()\n    ", "id": "1381859"}, "references": ["\nhttps://lists.x.org/archives/xorg-announce/2016-October/002720.html\n    "], "details": ["\nThe XGetImage function in X.org libX11 before 1.6.4 might allow remote X servers to gain privileges via vectors involving image type and geometry, which triggers out-of-bounds read operations.\n    "], "upstream_fix": "libX11 1.6.4", "cvss": {"cvss_scoring_vector": "AV:A/AC:M/Au:N/C:N/I:N/A:P", "status": "draft", "cvss_base_score": "2.9"}}